tests/bluetooth/tester/src/btp_gatt.c

```diff
@@ -483,8 +487,6 @@ static void add_characteristic(uint8_t *data, uint16_t len)
 		   CONTROLLER_INDEX, BTP_STATUS_FAILED);
 }
 
-static bool ccc_added;
-
 static uint8_t ccc_value;
```

Comment:
ccc_value is shared ,  so I think those (ccc_value and ccc_added) should be per characteristic (maybe in user data 'struct gatt_value'?)

Reply:
yes, at least `ccc_value` (we add these descriptors and set characteristic values in sequence, so most recently added characteristic is updated). And hence `user_data` of CCC is not used I think it's good place to add this